module1/homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,22 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence == 0 || sequence == 1)
+        return sequence;
+
+    int previousPreviousValue, previousValue{0}, currentValue{1};
+
+    for(std::size_t i = 1; i < sequence; ++i) {
+        previousPreviousValue = previousValue;
+        previousValue = currentValue;
+        currentValue = previousPreviousValue + previousValue;
+    }
+    return currentValue;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence == 0 || sequence == 1)
+        return sequence;
+
+    return (fibonacci_recursive(sequence-2) + fibonacci_recursive(sequence-1));
```

Comment:
If you are using whitespaces around operators, use them also here 
```suggestion
    return (fibonacci_recursive(sequence - 2) + fibonacci_recursive(sequence - 1));
```